### PR TITLE
add routegroups to reports

### PR DESCRIPTION
**Pull request by szuecs**
Mostly implemented analog to ingress without status checks

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -120,6 +120,19 @@ def fake_responses():
                 }
             ]
         },
+        ("routegroups.zalando.org/v1", "routegroups"): {
```
Comment (hjacobs) [inline tests/conftest.py:123]: I think the first tuple entry (API group version) is wrong and does not need the "routegroups"

**Comment by hjacobs** [comment]
Thanks! Can you fix the build (`make lint`)?

**Comment by szuecs** [comment]
```
% make lint
poetry install
make: poetry: Command not found
Makefile:11: recipe for target 'install' failed
make: *** [install] Error 127
zsh: exit 2     make lint
```
pip3 install poetry seems to do nothing...

**Comment by hjacobs** [comment]
Replaced by #178

**Pull request merged**